bitly/sprinterns2024
# about us page

created the skeleton for the about us page. 

To test: 
1) `go run main.go`
2) `cd views/js/evently`
3) `npm start`
This is what the page should look like: 
<img width="943" alt="image" src="https://github.com/bitly/sprinterns2024/assets/117085136/ad50de3b-c1e9-43b5-9bb7-cf06d9753b37">
<img width="941" alt="image" src="https://github.com/bitly/sprinterns2024/assets/117085136/09cd454b-0bbc-4098-a8f9-7f2c20f201fb">
<img width="933" alt="image" src="https://github.com/bitly/sprinterns2024/assets/117085136/0c08e67a-e055-401f-ab65-fcf841412fee">
